articles/api/management/v2/user-search/v3/index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: User Search
+description: Learn about Auth0's user search query string syntax and how to search for users then sort the results.
+toc: true
+---
+
+# User Search v3
+
+Looking for someone? Auth0's [list or search users](/api/management/v2#!/Users/get_users) endpoint enables you to search for users using [Lucene Query Syntax](http://www.lucenetutorial.com/lucene-query-syntax.html), select the fields to return, and sort the results.
+
+In this article you'll learn about the query string syntax, how to search for users, and how to sort the results.
+
+## Before you start
+
+* If you are using our [user search engine v2](/api/management/v2/user-search/v2), check out the [section on migrating from v2 to v3](#migrate-from-search-engine-v2-to-v3) below.
+* You'll need a token to make requests to the Management API. Check out [the Auth0 Management APIv2 token](/api/management/v2/tokens) for more information.
+* To perform user search requests the `read:users` [scope](/scopes/current) is required.
+
+## Search for users
+
+To search for users, make a `GET` request to the [`/api/v2/users` endpoint](/api/management/v2#!/Users/get_users). Pass your search query in query string syntax to the `q` parameter and set the `search_engine` parameter to `v3`.
+
+### Example Request
+
+For example, to search for users whose email is exactly "jane@exampleco.com", use `q=email:"jane@exampleco.com"`:
```

Comment:
use `this` for values (i.e. whose email is exactly `jane@exampleco.com`)

Reply:
The plural also confused me a bit since you use email in your example (isn't it unique?)